src/SurveyService.Validation/Question/CreateQuestionRequestValidator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question.Filters;
+using LT.DigitalOffice.SurveyService.Validation.Option.Interfaces;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Question;
+
+public class CreateQuestionRequestValidator : AbstractValidator<CreateQuestionRequest>, ICreateQuestionRequestValidator
+{
+  public CreateQuestionRequestValidator(
+    IQuestionRepository _questionRepository
+    )
+  {
+    RuleFor(q => q.Content)
+      .MaximumLength(300)
+      .WithMessage("Content is too long.");
+
+    When(q => q.Deadline.HasValue, () =>
+    {
+      RuleFor(q => q.Deadline)
+        .Must(v => v > new DateTime(v.Value.Year, v.Value.Month, v.Value.Day + 1,
```

Comment:
v changed to d. DateTime.UtcNow.AddDays(1) may not work if at least a millisecond has passed since the request was created. I changed to "d => d > DateTime.UtcNow.AddDays(1).AddSeconds(2)", but I'm still not sure if that will work.  The conversation is still open, maybe marina can tell me how to do it right

Reply:
Done